eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/actions/ShareActionExecutable.java

```diff
@@ -60,10 +62,24 @@ public void execute(@NonNull Context context)
 
         String eventText = eventText(context);
 
-        Intent sendIntent = new Intent();
-        sendIntent.setAction(Intent.ACTION_SEND);
-        sendIntent.putExtra(Intent.EXTRA_TEXT, eventText);
-        sendIntent.setType("text/plain");
+        Intent sendIntent;
+        if (context instanceof Activity)
```

Comment:
I thought about action buttons on Notifications as a theoretical possibility.
For example getting directions could be a button on a notification.